### ISSUE TITLE
Error while running: dox-foundation.js:32 - files Object has no method 'map'

### ISSUE THREAD

**Issue by airandfingers** [mattmcmanus/dox-foundation#17]
``` bash
$ dox-foundation < Desktop.js > desktop_dox.html

/usr/lib/node_modules/dox-foundation/lib/dox-foundation.js:32
    options.structure = files.map(function(file){
                              ^
TypeError: Object #<Object> has no method 'map'
    at Object.exports.parse (/usr/lib/node_modules/dox-foundation/lib/dox-foundation.js:32:31)
    at null.<anonymous> (/usr/lib/node_modules/dox-foundation/bin/dox-foundation:96:30)
    at EventEmitter.emit (events.js:85:17)
    at afterRead (fs.js:1324:12)
    at Object.wrapper [as oncomplete] (fs.js:362:17)
```

**Comment by mattmcmanus** [comment]
This is definitely a result of some changes I made recently. I've been focusing on the folder parsing and haven't been checking the stdio functionality. I'll try and fix this as soon as I can.

**Comment by mattmcmanus** [comment]
Alrighty, I fixed this in 448d679e3790e3dce14db9261e638baea8c2f546. I'll push out a new version to npm shortly
